_posts/databases/postgresql/2000-01-01-start.md

```diff
@@ -248,7 +273,36 @@ ALTER DEFAULT PRIVILEGES FOR USER <database> IN SCHEMA public GRANT SELECT ON SE
 
 {% include database_backups.md %}
 
-### Download automated backups
+Scalingo PostgreSQL databases offer two way of backing up the data.
+
+### Point-in-Time Recovery
+
+Point-in-time recovery (PITR) allows you to ask for the restoration of your data
+at a specific date. We achieve this by making a full PITR backup of the database
+weekly. Between two PITR backups, we keep track of the write-ahead logs (WAL).
+The WAL contains all the modification instructions. By replaying the WAL from a
+PITR backup to a specific date, we are able to rebuild the state of the database
+at that date.
+
+### On-demand Backups
```

Comment:
can you link the main paragraph explaining all the subtleties of the new backup procedure?